webvtt.html

```diff
@@ -1171,7 +1168,7 @@
     contain the substring "<code>--></code>" (U+002D HYPHEN-MINUS, U+002D HYPHEN-MINUS, U+003E
     GREATER-THAN SIGN).</p>
 
-    <p>A <dfn>WebVTT cue</dfn> consists of the following components, in the given order:</p>
```

Comment:
This concept is needed, it's referenced from "WebVTT file body" which is now pointing to the wrong definition.